src/screens/FeedScreen/elements/BlurredLayer.js

```diff
@@ -99,7 +99,7 @@ const styles = StyleSheet.create({
     fontWeight: 'normal',
     fontSize: normalizeFontSizeByWidth(12),
     lineHeight: 18,
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color change from `COLORS.white` to `COLORS.almostBlack` might affect the visibility of the text depending on the background color. Please ensure that this color change is in line with the overall design and doesn't affect readability.
```diff
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->